src/core/config.py

```diff
@@ -8,6 +8,8 @@ class Config:
     GEMINI_MODEL = os.environ.get('GEMINI_MODEL', 'gemini-1.5-flash-002')
     OPENAI_API_KEY = os.environ.get('OPENAI_API_KEY')
     OPENAI_MODEL = os.environ.get('OPENAI_MODEL', 'gpt-4o-mini')
+    ANTHROPIC_API_KEY = os.environ.get('ANTHROPIC_API_KEY')
+    ANTHROPIC_MODEL = os.environ.get('ANTHROPIC_MODEL', 'claude-3-5-sonnet-latest')
 
```

Comment:
Bạn nên thêm kiểm tra xem `ANTHROPIC_API_KEY` có được thiết lập hay không. Nếu không, ứng dụng có thể gặp lỗi khi cố gắng sử dụng Anthropic API.  Hãy xem xét thêm một đoạn code để xử lý trường hợp này, ví dụ như ném một exception hoặc sử dụng một giá trị mặc định an toàn.  Ví dụ:
```python
    if not ANTHROPIC_API_KEY:
        raise ValueError("ANTHROPIC_API_KEY environment variable is not set")
``` hoặc
```python
    ANTHROPIC_API_KEY = os.environ.get('ANTHROPIC_API_KEY', None) 
``` và xử lý trường hợp `None` sau đó.